src/main/java/mate/controller/car/CarDeleteController.java

```diff
@@ -0,0 +1,22 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+public class CarDeleteController extends HttpServlet {
```

Comment:
<img width="930" alt="image" src="https://user-images.githubusercontent.com/72727656/202678221-18ac1f6d-dcf9-4d97-9aeb-278c1efcf260.png">

```suggestion
public class DeleteCarController extends HttpServlet {
```